packages/sdk/src/modules/liquidation/getPotentialLiquidation.ts

```diff
@@ -71,8 +77,20 @@ export default async function getPotentialLiquidation(
     .mul(closeFactor)
     .div(BigNumber.from(10).pow(borrower.debt[0].underlyingDecimals.toNumber()));
 
+  const protocolFee = liquidationAmount.mul(PROTOCOL_FEE).div(1000);
```

Comment:
you need to fetch it from the market as mantissa (scaled by 1e18) and add it to the liquidation incentive like it is done here https://github.com/Midas-Protocol/contracts/blob/b7fec7f083ca2e69d94bbdab2e02ac7742be2f3a/contracts/compound/Comptroller.sol#L923